source/fuzz/fuzzer_context.cpp

```diff
@@ -38,6 +38,8 @@ const std::pair<uint32_t, uint32_t> kChanceOfAddingLocalVariable = {20, 90};
 const std::pair<uint32_t, uint32_t> kChanceOfAddingMatrixType = {20, 70};
 const std::pair<uint32_t, uint32_t> kChanceOfAddingNoContractionDecoration = {
     5, 70};
+const std::pair<uint32_t, uint32_t> kChanceOfAddingParameters = {
+    5, 70};  // Adjust this
```

Comment:
Seems reasonable.  At some point we should do some empirical work to make these guesses more informed.

Reply:
Removed comment in 167cc6e.